### DISCUSSION TITLE
Debug with Pycharm IDE

### DISCUSSION THREAD

**Discussion by jpsteil** [emmett-framework/emmett#403] [Q&A] [answered]
Trying to setup Pycharm to use the interactive debugger with Emmett.  I'm setting up this command to run to start Emmett

python3 -m emmett -a appname develop

But get this error:

Traceback (most recent call last):
  File "/usr/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/jim/.local/lib/python3.9/site-packages/emmett/__main__.py", line 13, in <module>
    from .cli import main
ImportError: attempted relative import with no known parent package

Is there a way to start the Emmett server from Python?  Or, am I going about this all wrong?

**Comment by gi0baro** [comment, marked as answer]
@jpsteil unfortunately seems that the reloader is messing a bit with imports. The `develop` command works using the `-m` notation only with `--no-reloader` option.

I suggest you to use directly the `emmett develop` command from a virtual environment.

**Comment by jpsteil** [comment]
Thanks for the reply!

-Jim
